### ISSUE TITLE
backbones/renet.py

### ISSUE THREAD

**Issue by 1303463822** [leondgarse/Keras_insightface#111]
in resnet.py, the ResNet101 uses basicblock with depthes = [3, 4, 23, 3] rather than bottleneck. Are there any errors?

**Comment by leondgarse** [comment]
Not error, just following the usage in official insightface, and it's accuracy is bit higher. You can also find some training log using typical ResNet50V2/ResNet100V2 in [Resnet50V2 / Resnet101V2 swish using SGD + L2 regularizer + cosine lr decay training on MS1MV3 dataset](https://github.com/leondgarse/Keras_insightface/discussions/15#discussioncomment-790754). Or just use `keras.applications.ResNet101` by using `"resnet101"` rather than `"r100"` when initializing `basic_model`, built by [models.py#L39-L45](https://github.com/leondgarse/Keras_insightface/blob/master/models.py#L39-L45).
```py
import models
basic_model = models.buildin_models("resnet101", dropout=0, emb_shape=512, output_layer='E', bn_momentum=0.9, bn_epsilon=1e-5)
...
```
